packages/node/src/node.ts

```diff
@@ -1 +1,77 @@
-export default class Node {}
+import * as cf from "@counterfactual/cf.js";
+import machine from "@counterfactual/machine";
+import * as ethers from "ethers";
+import { EventEmitter } from "eventemitter3";
+import firebase from "firebase";
+
+export class Node extends EventEmitter {
```

Comment:
@cf19drofxots If it's the only class in the file, should we use `export default`?